common.go

```diff
@@ -0,0 +1,113 @@
+package appcenter
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"mime/multipart"
+	"net/http"
+	"net/http/httputil"
+	"os"
+)
+
+const (
+	baseURL = `https://api.appcenter.ms`
+)
+
+func (c Client) jsonRequest(method, url string, body interface{}, response interface{}) (int, error) {
```

Comment:
Is some kind of retry logic possible?

Reply:
retry logic sounds like a requirement for client side implementation